resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -294,3 +294,13 @@ In addition, you can run a single test selectively.
         -Dspark.kubernetes.test.deployMode=docker-desktop \
         -Dspark.kubernetes.test.imageTag=2022-03-06 \
         'kubernetes-integration-tests/testOnly -- -z "Run SparkPi with a very long application name"'
+
+You can also specify your specific dockerfile to build JVM/Python/R based image to test.
+
+    build/sbt -Psparkr -Pkubernetes -Pkubernetes-integration-tests \
+        -Dtest.exclude.tags=minikube \
+        -Dspark.kubernetes.test.deployMode=docker-desktop \
+        -Dspark.kubernetes.test.DockerFile=/path/to/Dockerfile \
```

Comment:
Is this tested, @Yikun ? It looks wrong because the correct one is `spark.kubernetes.test.dockerFile`.

Reply:
I have to say sorry, I only tested it with new comming configurations. will do a complete test.